frontend/tests/e2e_tests/docker-compose.e2e-tests.yml

```diff
@@ -1,25 +1,27 @@
+name: gui-tests
 services:
-  mender-gui-tests-runner:
+  gui-tests-runner:
     image: mendersoftware/mender-test-containers:gui-e2e-testing
     command: tail -f /dev/null
```

Comment:
If this service is run-only you can set `scale: 0`.

Reply:
does it make a difference if this is omitted?